src/components/Details.css

```diff
@@ -0,0 +1,74 @@
+/* .login-container {
```

Comment:
- [ ] Please, kindly delete this old piece of commented code, to improve the cleanliness of the app. 💪